gettsim/transfers/kinderbonus.py

```diff
@@ -8,17 +7,14 @@ def kinderbonus_m(
 
     Parameters
     ----------
-    kinderfreib_günstiger_tu
-        See :func:`kinderfreib_günstiger_tu`.
     kinderbonus_basis_m
         See :func:`kinderbonus_basis_m`.
 
     Returns
     -------
 
     """
-    out = 0.0 if kinderfreib_günstiger_tu else kinderbonus_basis_m
```

Comment:
It seems that `kinderbonus_m` can be removed and `kinderbonus_basis_m` can be renamed to `kinderbonus_m`. Same for kindergeld

Reply:
Yes, absolutely!